tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    django22-py{36,37,38}-drf{39,310,311}-{sqlite,pg}
+    django{22,3}-py{36,37,38}-drf311-{sqlite,pg}
```

Comment:
Here we dropped drf 3.9 and 3.10. Was it really necessary? Plus there is a deprecation process, we can't just drop the support out of the blue.